_pages/print.html

```diff
@@ -0,0 +1,23 @@
+---
+permalink: /print
+title: Print
+layout: none
+excerpt: none
+---
+
+<html lang="en">
+  {% include head.html %}
+  <link rel="stylesheet" type="text/css" href="{{site.baseurl}}/stylesheets/print.css">
+  <body class="layout-print">
+    {% for page in site.pages %}
+      {% unless page.title == "Introduction" %}
+        <div class="article">
```

Comment:
There's probably an underlying reason I'm not aware of, but why `.article` rather than an `article` element?